rust/src/external/ntp.rs

```diff
@@ -0,0 +1,396 @@
+/* Copyright (C) 2017 Open Information Security Foundation
```

Comment:
wonder if we should make this rust/src/external/ntp/ntp.rs

With the built-in parsers we have a directory for all the parser components: nom parsers, log, detect, etc.